spec/features/step_by_step_actions_spec.rb

```diff
@@ -22,14 +27,39 @@
   end
 
   context "Step by step has been submitted for 2i" do
-    scenario "show the relevant actions to the step by step author" do
+    background do
       given_there_is_a_step_by_step_that_has_been_submitted_for_2i
+    end
+
+    scenario "show the relevant actions to the step by step author" do
       and_I_am_the_step_by_step_author
       when_I_visit_the_step_by_step_page
       then_there_should_be_no_primary_action
       and_the_secondary_action_should_be "Preview"
       and_there_should_be_tertiary_actions_to %w(Delete)
     end
+
+    scenario "show the relevant actions to the step by step 2i reviewer" do
+      and_I_am_the_reviewer
+      when_I_visit_the_step_by_step_page
+      then_the_primary_action_should_be "Claim for 2i review"
+      and_the_secondary_action_should_be "Preview"
+      and_there_should_be_tertiary_actions_to %w(Delete)
+    end
+  end
+
+  context "Step by step has been claimed for 2i" do
+    background do
+      given_there_is_a_step_by_step_that_has_been_claimed_for_2i
+    end
```

Comment:
I'd make `given_there_is_a_step_by_step_that_has_been_claimed_for_2i` part of the scenario below and only create a `background` block when it's needed.

Reply:
Moved as suggested, but from where I stand there will be multiple blocks rather soon.